### ISSUE TITLE
train/val curve

### ISSUE THREAD

**Issue by lsq357** [salman-h-khan/PL-ZSD_Release#13]
Hi! Can you show me the loss curve?
Or how many steps is better to train ?

**Comment by shafin5** [comment]
From my experience, if you use the pre-trained model, then 30 epochs with 10,000 steps per epoch should be enough for MS-COCO dataset.
